Introduction.adoc

```diff
@@ -1,26 +1,36 @@
+== Introduction
```

Comment:
This is redundant with respect to the title in https://github.com/zynaddsubfx/user-manual/blob/master/Manual.adoc